docs/security/openid-connect.md

```diff
@@ -280,6 +280,7 @@ opendistro_security.openid.client_secret | The client secret of the OpenID Conne
 opendistro_security.openid.scope | The [scope of the identity token](https://auth0.com/docs/scopes/current) issued by the IdP. Optional. Default is `openid profile email address phone`.
 opendistro_security.openid.header | HTTP header name of the JWT token. Optional. Default is `Authorization`.
 opendistro_security.openid.logout_url | The logout URL of your IdP. Optional. Only necessary if your IdP does not publish the logout URL in its metadata.
+opendistro_security.openid.base_redirect_url | The base of the redirect URL that will be sent to youe IdP. If unspecified, kibana will use the server.host and server.port. Useful when kibana is behind a reverse proxy.
```

Comment:
There's a small typo here: `to youe IdP`

Reply:
thx. I did rephrase.